packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2954,6 +2959,7 @@ export class ContainerRuntime
 			node: { type: "DataStore", path: `/${internalId}` },
 			reason: "Loaded",
 			packagePath: context.packagePath,
+			timestampMs: this.getCurrentReferenceTimestampMs(),
```

Comment:
nit: consider pulling into separate PR

Reply:
You are right. This is left over from the change that I started doing but then pivoted to this.